ci/travis/ci.sh

```diff
@@ -253,10 +258,16 @@ install_ray() {
   (
     cd "${WORKSPACE_DIR}"/python
     build_dashboard_front_end
-    keep_alive pip install -v -e .
+    keep_alive pip install -v -v -e .
```

Comment:
What does `-v -v` do?

Reply:
Oops this is left over from debugging, meant to take it out.